pennylane/transforms/mitigate.py

```diff
@@ -318,6 +318,47 @@ def richardson_extrapolate(x, y):
     return poly_extrapolate(x, y, len(x) - 1)
 
 
+def exponential_extrapolate(x, y, asymptote=None, eps=1.0e-6):
+    r"""Extrapolate to the zero-noise limit using an exponential model (:math:`Ae^{Bx} + C`). This
+    is done by linearizing the data using a logarithm, whereupon a linear fit is performed. Once
+    the model parameter are found, they are transformed back to exponential parameters.
+
+    Args:
+        x (Array): Data in x
+        y (Array): Data in y = f(x)
+        asymptote (float): Infinite noise limit expected for your circuit of interest. Defaults to
```

Comment:
For clarity, should we mention that this is the `C` you introduced in the equation above?